sos/report/plugins/system.py

```diff
@@ -23,6 +23,14 @@ def setup(self):
             "/etc/sysconfig",
             "/etc/default",
             "/etc/environment",
+            "info -o vd",
+            "info -o pd",
+            "info -o hba",
```

Comment:
The `info` command is traditionally a man-like command, and in particular the `-o` option pushes output to a file, which definitely would not make sense in an sos plugin.

Does `info` map to a different command on powerpc? If so, I'd recommend these go in the `powerpc` plugin, not `system.

Reply:
These look like Marvell mvCLI command verbs (`-o vd`, `-o pd` and `-o hba` are all valid arguments to the `mvcli info` command, and nothing else that I know).

These adapters are common on IBM systems but if that's the case then they won't work without (at least) the `mvcli` command as prefix - it's also possibly common that the binary is installed in a non-standard bin directory, meaning that we would either need to provide an absolute path or modify the pre-set PATH used by `sos`.

More information on the commands can be found [here](https://www.ibm.com/support/knowledgecenter/9040-MR9/p9ej8/p9ej8_recreate_drive.htm) and [here](https://www.ibm.com/support/knowledgecenter/TI0003K/p8eip/p8eip_drive_commands_mvcli.htm).

Reply:
So yes: definitely not approariate for the system plugin. These need to find a new home, either in a plugin of their own or one of the existing plugins that's more specific to the systems where these devices are found.

Reply:
Oh, also I didn't notice this before, but these are also in the wrong call here as well. Command output needs to be collected via `add_cmd_output()`, not `add_copy_spec()`